Makefile

```diff
@@ -290,7 +290,9 @@ gen-api: gen-apiserver-api gen-uibackend-api ## Generating API code
 .PHONY: gen-apiserver-api
 gen-apiserver-api: ## Generating Go library for API specification
 	$(info Generating API for backend code ...)
-	@(cd api && go generate)
+	@(cd api/client && go generate)
+	@(cd api/server/pkg && go generate)
+	@(cd api/types && go generate)
```

Comment:
Pls make `@(cd api/types && go generate)` first in the list as others depend on it's output.